content/en/blog/2022/prow-and-tide-for-kubernetes-contributors/prow-and-tide-for-kubernetes-contributors.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: blog
+title: Prow and Tide for Kubernetes Contributors
+date: 2022-05-31
+slug: prow-and-tide-for-kubernetes-contributors
+---
+
+**Authors:** [Chris Short](https://github.com/chris-short), [Frederico Muñoz](https://github.com/fsmunoz)
+
+---
+
+In my work in the Kubernetes world, I look up a label or Prow command often. The systems behind the scenes ([Prow](https://prow.kubernetes.io/) and [Tide](https://pkg.go.dev/k8s.io/test-infra/prow/cmd/tide#section-readme)) are here to help Kubernetes Contributors get stuff done.
+
+Labeling which SIG, WG, or subproject is as important as the issue or PR having someone assigned. To quote [the docs](https://docs.prow.k8s.io/docs/components/core/tide/), "Tide is a [Prow](https://docs.prow.k8s.io/docs/) component for managing a pool of GitHub PRs that match a given set of criteria. It will automatically retest PRs that meet the criteria ('tide comes in') and automatically merge them when they have up-to-date passing test results ('tide goes out')."
+
+What actually prompted this article is the awesomely amazing folks on the Contributor Comms team saying, "I need to squash my commits and push that." Which immediately made me remember the wonder of the Tide label: [`tide/merge-method-squash`](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#tide/merge-method-squash).
```

Comment:
Should we link to the Contributor Comms GH repo so folks know where to find us?

Reply:
Done.